mrcnn/model.py

```diff
@@ -2796,19 +3179,55 @@ def parse_image_meta_graph(meta):
     }
 
 
-def mold_image(images, config):
+def resnet_mold_image(images, config):
     """Expects an RGB image (or array of images) and subtracts
     the mean pixel and converts it to float. Expects image
     colors in RGB order.
     """
     return images.astype(np.float32) - config.MEAN_PIXEL
 
 
-def unmold_image(normalized_images, config):
-    """Takes a image normalized with mold() and returns the original."""
+def resnet_unmold_image(normalized_images, config):
+    """Takes a image normalized with resnet_mold() and returns the original."""
     return (normalized_images + config.MEAN_PIXEL).astype(np.uint8)
 
 
+def mobilenet_mold_image(images, config):
+    """Expects an RGB image (or array of images) and normalizes by
+    dividing by the median pixel value and subtracting 1.
+    """
+    return images.astype(np.float32)/127.5 - 1.0
```

Comment:
No need to create a separate function. Both are normalizing the image, just with different values. Update MEAN_PIXEL, and add config.NORMALIZE_PIXEL such that you can control the normalization in configuration without adding new functions.

Reply:
In fact, @JonathanCMitchell 's MobileNetV2 [implementation](https://github.com/keras-team/keras-applications/blob/master/keras_applications/mobilenet_v2.py) uses `imagenet_utils.preprocess_input` like the resnet50 implementation so I don't think there will be any change from the resnet mold/unmold. However, isn't `images.astype(np.float32)/127.5 - 1.0` based on the batch and `config.MEAN_PIXEL` based on the whole imagenet dataset?